### PR TITLE
Init commit

### PR DESCRIPTION
**Pull request by klzns**
Implements OneSignal integration:
https://onesignal--storecomponents.myvtex.com/

Scroll the page a bit to see the notification to accept notifications.

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "projectName": "react-app-template",
```
Comment (Klynger) [inline .all-contributorsrc]: missing change

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#### What does this PR do? \*
```
Comment (iaronaraujo) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: I think this is the old template, but we can update this later

**Review by iaronaraujo — APPROVED** [review]

File: node/package.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  "dependencies": {
+    "co-body": "^6.0.0",
+    "ramda": "^0.25.0"
+  },
+  "devDependencies": {
+    "@vtex/tsconfig": "^0.4.4",
+    "@types/co-body": "^0.0.3",
+    "@types/jest": "^24.0.18",
+    "@types/node": "^12.0.0",
+    "@types/ramda": "types/npm-ramda#dist",
```
Comment (Klynger) [inline node/package.json]: What's this?

**Review by Klynger — CHANGES_REQUESTED** [review]

File: node/package.json (hunk 3 above)
Comment (klzns) [reply node/package.json]: I actually don't know. It came with the project service-example that I used as a starting point

**Review by Klynger — APPROVED** [review]

**Pull request merged**